src/v2/ion/i18nTransformer.js

```diff
@@ -374,22 +374,15 @@ const updateLabelForUiSchema = (remediation, uiSchema) => {
 
 };
 
-const isWebAuthnAPIError = ( i18nKey ) => i18nKey.startsWith(WEBAUTHN_API_GENERIC_ERROR_KEY);
+const isWebAuthnAPIError = (i18nKey) => i18nKey.startsWith(WEBAUTHN_API_GENERIC_ERROR_KEY);
 
-/**
- * @typedef {Object} Message
- * @property {string} message
- * @property {Object=} i18n
- * @property {string} i18n.key
- * @property {string[]} i18n.params
- */
 /**
  * - If `message.i18n.key` exists and has a value in 'login.properties'
  *   through the given key or via I18N_OVERRIDE_MAPPINGS, return the value.
  *
  * - returns `message.message` otherwise
  *
- * @param {Message} message
```

Comment:
The jsDoc was a bit incorrect, while they tried to make i18n an object, it was not being recognized as such, so I just used the import for the param to re-use the IdxMessage interface instead of re-defining a new one.

Reply:
maybe keep those doc comments unchanged for now even it's not fully correct. mostly the concern is all these changes will be questioned again when merge 0.0 to master.

Reply:
When I leave the doc comments the same, it causes an TS error when we use them, I will just use a @ts-ignore in the location where we use this function to not modify v2 files.

Reply:
oh, ic, then keep it updated